src/machine/machine_rp2040_clocks.go

```diff
@@ -14,6 +14,10 @@ const (
 	MHz = 1000000
 )
 
+func CPUFrequency() uint32 {
+	return 125 * MHz
```

Comment:
According to the datasheet (https://datasheets.raspberrypi.org/rp2040/rp2040-datasheet.pdf), it's (up to) 133MHz.
I didn't write the code, so I don't know what frequency it's actually running at.

See: https://github.com/tinygo-org/tinygo/pull/1897, CC @rythmkraze.

Reply:
In MicroPython it runs at 125 MHz by default, but you can set it as high as 270 MHz (and this applies to both cores). Probably not recommended for long term use though.